### PR TITLE
Add title tag

### PR DESCRIPTION
**Pull request by jeremykenedy**
Added title tag to toggle attr, show, and hidden states.

### CODE REVIEW

**Comment by tylersticka** [comment]
@jeremykenedy Thanks for contributing this PR!

It would be helpful to have a little more context for these changes. Currently, the button has an `aria-label` of "Show Password" and the `aria-pressed` attribute is toggles to `true` or `false`. My concern with these changes is that if the `title` is ever set to `Hide Password` while `aria-pressed` is `true`, that it will seem as if clicking the button will do the opposite of what it does.

But if this was change was fueled by actual usability issues you encountered while using accessibility technologies, we definitely want to hear about it!

**Comment by jeremykenedy** [comment]
I introduced the title tag due to when only using the icon for toggle there is not indicator as to what it does other than the icon, with the title tag it will give you the "hide" and "show" verbiage on hover.

File: hideShowPassword.js [hunk 1]
```diff
@@ -112,6 +112,7 @@
       attr: {
         role: 'button',
         'aria-label': 'Show Password',
+        'title': 'Show Password',
```
Comment (tylersticka) [inline hideShowPassword.js]: Unquote `'title'` (legal prop name does not require quotes).

File: hideShowPassword.js [hunk 2]
```diff
@@ -162,7 +163,10 @@
         toggle: {
           className: 'hideShowPassword-toggle-hide',
           content: 'Hide',
-          attr: { 'aria-pressed': 'true' }
+          attr: { 
+            'aria-pressed': 'true',
+            'title': 'Hide Password'
```
Comment (tylersticka) [inline hideShowPassword.js]: Unquote `'title'` (legal prop name does not require quotes).

File: hideShowPassword.js [hunk 3]
```diff
@@ -172,7 +176,10 @@
         toggle: {
           className: 'hideShowPassword-toggle-show',
           content: 'Show',
-          attr: { 'aria-pressed': 'false' }
+          attr: { 
+            'aria-pressed': 'false',
+            'title': 'Show Password'
```
Comment (tylersticka) [inline hideShowPassword.js]: Unquote `'title'` (legal prop name does not require quotes).

**Comment by tylersticka** [comment]
@jeremykenedy Thanks for responding to my question!

I had to really think this over. Initially I was skeptical because an icon-only treatment is an implementation choice on the part of the developer, and the code already allows the implementor to pass along custom attributes for each state (though the syntax for doing so is pretty verbose).

That said, the alternative text for this is simply "Show" and "Hide" within the element itself, which may not be enough to clue user's in to what is actually going on. So as of this morning I'm coming around to this being a nice default to implement.

I've made a few small formatting nits inline. If you wouldn't mind making those adjustments, I'd be happy to merge this in and give it a version bump.

**Comment by jeremykenedy** [comment]
Hello, thanks for letting me contribute :)  I have made the changes. Cheers!

**Comment by tylersticka** [comment]
Thanks, @jeremykenedy! Your changes are in version `2.0.9`.

**Pull request merged**